crates/sui-network/src/state_sync/server.rs

```diff
@@ -96,6 +101,26 @@ where
         Ok(Response::new(checkpoint))
     }
 
+    async fn get_peer_latest_checkpoint_info(
+        &self,
+        _request: Request<()>,
+    ) -> Result<Response<GetPeerLatestCheckpointInfoResponse>, Status> {
+        let highest_synced = self
```

Comment:
optional nit- make these variable names match the struct field names, and then you don't have to write names twice in the struct initializer below?